diff --git test/BUILD.bazel test/BUILD.new.bazel
index 28a1eab..bb78974 100644
--- test/BUILD.bazel
+++ test/BUILD.new.bazel
@@ -83,18 +83,16 @@ $(execpath  //:gawk) -f $(execpath {AWK}.awk) < {IN} > $@ || echo "EXIT CODE: $$
     for awk in AWK_TESTS
 ]
 
+TODO_TESTS = [
+    "apiterm",  # `@load "filefuncs"` not modelled in genrule deps.
+]
+
 [
     diff_test(
         name = awk + "_diff_test",
         file1 = awk + "_gen.out",
         file2 = awk + ".ok",
+	tags = ["manual"] if awk in TODO_TESTS else [],
     )
     for awk in AWK_TESTS
-    if awk in OK and awk not in [
-        # TODO: These tests are not configured correctly.
-        "aadelete2",
-        "apiterm",
-        "ar2fn_elnew",
-        "ar2fn_elnew_sc",
-    ]
 ]
