diff --git test/BUILD.bazel test/BUILD.bazel
index 23271c8..2a8eba8 100644
--- test/BUILD.bazel
+++ test/BUILD.bazel
@@ -89,12 +89,30 @@ TODO_TESTS = [
     "apiterm",  # `@load "filefuncs"` not modelled in genrule deps.
 ]
 
+# Tests broken on Windows for unknown reasons; diff_test does not actually print the diff when it fails on that platform.
+TODO_WINDOWS_TESTS = [
+    "aadelete1",
+    "aadelete2",
+    "aarray1",
+    "aasort",
+    "aasorti",
+    "addcomma",
+    "anchgsub",
+    "anchor",
+    "ar2fn_elnew_sc",
+]
+
 [
     diff_test(
         name = awk + "_diff_test",
         file1 = awk + "_gen.out",
         file2 = awk + ".ok",
         tags = ["manual"] if awk in TODO_TESTS else [],
+        target_compatible_with = select({
+            "@platforms//os:osx": [],
+            "@platforms//os:linux": [],
+            "@platforms//os:windows": ["@platforms//:incompatible"] if awk in TODO_WINDOWS_TESTS else [],
+        }),
     )
     for awk in AWK_TESTS
 ]
